<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormMenu</class>
 <widget class="QDialog" name="FormMenu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>258</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>258</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>258</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>功能菜单</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>210</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::NoButton</set>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonOpenTemplateDir">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>151</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>打开模板目录</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonImportTemplate">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>151</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>导入自定义模板</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonCustomTemplate">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>151</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>编辑自定义模板</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonSyncTemplates">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>10</y>
     <width>151</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>从云端同步模板</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonCheckUpdate">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>70</y>
     <width>151</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>检查更新</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonDebug">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>190</y>
     <width>151</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>打开调试输出</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>FormMenu</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>FormMenu</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
